views/index.hbs

```diff
@@ -15,14 +15,8 @@
 			document.documentElement.className = document.documentElement.className.replace(/\bcore\b/g, 'enhanced');
 		}
 	</script>
+	<link rel="stylesheet" href="/static/build/inline.css" type="text/css" />
```

Comment:
This is technically an external stylesheet, inline would be if you used a style tag.